website/docs/r/cluster.html.markdown

```diff
@@ -43,7 +43,7 @@ resource "rancher2_cluster" "foo-custom" {
             max_size = 100
             path = "-"
             format = "json"
-            policy = jsonencode({"rules":[{"level": "Metadata"}]})
+            policy = jsonencode({"apiVersion":"audit.k8s.io/v1","kind":"Policy","rules":[{"level":"RequestResponse","resources":[{"group":"","resources":["namespaces"]}]}]})
```

Comment:
Can we do this another way, that feels kind of ugly. Should we recommend the user create a file and read in the yaml snippet?

Reply:
Agreed. Better now??

Reply:
Why are we reading JSON and not YAML? Writing JSON is not fun for a human.

Reply:
Agreed, but policy is expected with this format. Tell me if more changes are needed

Reply:
Ok, you win. Updated to yaml format. :)